server/controller/authController.js

```diff
@@ -4,10 +4,12 @@ const Restaurant = require("../collections/restaurant");
 module.exports = {
   login: (req, res, next) => {
     const { typedEmail, typedPassword } = req.body;
-    // console.log(typedEmail, typedPassword);
+    console.log(req.body)
```

Comment:
If it's being merged into master, you should probably comment out - or better, remove - extra unnecessary comments.